client/trino-jdbc/src/main/java/io/trino/jdbc/ConnectionProperties.java

```diff
@@ -478,6 +483,24 @@ public ExternalAuthentication()
         }
     }
 
+    private static class ExternalAuthenticationRedirectHandlers
+            extends AbstractConnectionProperty<List<ExternalRedirectStrategy>>
+    {
+        private static final Splitter ENUM_SPLITTER = Splitter.on(',').trimResults().omitEmptyStrings();
+
+        public ExternalAuthenticationRedirectHandlers()
+        {
+            super("externalAuthenticationRedirectHandlers", NOT_REQUIRED, ALLOWED, ExternalAuthenticationRedirectHandlers::parse);
```

Comment:
please pass the default value here and and in TrinoDriverUri, please simply expect always a value instead.

Reply:
There is already a default which is a Desktop open method. There is also a default for the CLI (print)